_sass/scale.scss

```diff
@@ -110,3 +110,22 @@ nav {
     font-size: .6em;
   }
 }
+
+/* Facebook button (news page & contact page) */
+.btn-fb {
+  font-size: 0.9rem;
+
+  /* responsive displays */
+  @media #{$mq-max-xsmall} {
+    font-size: 1rem !important;
```

Comment:
I suppose they're not needed anymore, while working on it I hadn't deleted the old styles yet... But I guess a simple test should tell us the answer...